gradle.properties

```diff
@@ -1,4 +1,4 @@
-version=29.34.3
+version=29.34.4
```

Comment:
This change removed a public builder, please bump minor version.

Reply:
Thanks for catching this. I have bumped the minor version. FWIW, I didn't see anyone using the public builder in codesearch.

Reply:
@nickibi Just to be clear: removing a public interface is a major version change, not a minor version change (however, it seems like we're electing to take the risk here since no one is apparently using this)